docs/reference/transform/usage.asciidoc

```diff
@@ -53,3 +53,13 @@ have a high level dashboard that is accessed by a large number of users and it
 uses a complex aggregation over a large dataset, it may be more efficient to
 create a {transform} to cache results. Thus, each user doesn't need to run the
 aggregation query.
+
+* You need to account for late-arriving data.
++
+In some cases, data might not be immediately available when a {transform} runs, leading to missing records in the destination index. This can happen due to ingestion delays, where documents take a few seconds or minutes to become searchable after being indexed.
+To handle this, the `delay` parameter in the {transform}'s sync configuration allows you to postpone processing new data. Instead of always querying the most recent records, the {transform} will skip a short period of time (e.g., 60 seconds) to ensure all relevant data has arrived before processing.
```

Comment:
```suggestion
To handle this, the `delay` parameter in the {transform}'s sync configuration allows you to postpone processing new data. Instead of always querying the most recent records, the {transform} will skip a short period of time (for example, 60 seconds) to ensure all relevant data has arrived before processing.
```